sympy/physics/units/definitions.py

```diff
@@ -358,7 +359,7 @@
 # Boltzmann constant
 boltzmann = boltzmann_constant = Quantity("boltzmann_constant")
 boltzmann_constant.set_dimension(energy/temperature)
-boltzmann_constant.set_scale_factor(1.38064852e-23*joule/kelvin)
+boltzmann_constant.set_scale_factor(1.380649e-23*joule/kelvin)
```

Comment:
Is there a reason this one is fewer digits now?

Reply:
In the latest update to the Boltzmann constant, it was set to be exactly that value. For reference: https://physics.nist.gov/cgi-bin/cuu/Value?k|search_for=boltzmann+constant .

Reply:
That URL just loads a blank page for me.

Reply:
Huh, strange. It's also in the [pdf version](https://physics.nist.gov/cuu/pdf/wallet_2018.pdf) that I linked in the issue.

Reply:
If it is an exact value then it should be entered as `Rational('1.380649', 10**23)`

Reply:
That is an interesting aspect. Most likely the numbers are given as decimal numbers, so the floating-point representation is not considered. Considering that there are additional uncertain digits, I'm not sure if it makes a huge difference though.

Reply:
They're not exact per se, since it's only an approximately measured value. Although some of the unit conversions are in fact exact. For example, the number of feet in a meter is exactly 0.3048, and it does use Rational https://github.com/sympy/sympy/blob/df9965b123b8141a4c75151beeb8db7620c774e9/sympy/physics/units/definitions.py#L216

I think we should keep using float here, or Float if the values have more precision than a Python float allows. At any rate, however, this should be a separate issue from this PR, which is just updating some values.